baxter_gazebo/worlds/baxter.world

```diff
@@ -11,11 +11,12 @@
 
     <physics type="ode">
       <real_time_update_rate>1000.0</real_time_update_rate>
-      <gravity>
-	0.0 0.0 -9.81
-      </gravity>
     </physics>
 
+    <gravity>
+      0.0 0.0 -9.81
+    </gravity>
```

Comment:
gravity is now a member of world, not physics

http://sdformat.org/spec?ver=1.6&elem=world#world_gravity